packages/appstore/src/components/modals/account-type-modal/account-type-modal.tsx

```diff
@@ -92,18 +92,36 @@ const MT5AccountTypeModal = () => {
     const { trading_platform_available_accounts } = client;
     const { enableApp, disableApp } = ui;
     const { setAppstorePlatform } = common;
+
+    React.useEffect(() => {
+        if (!is_account_type_modal_visible) {
+            selectAccountTypeCard('');
+        }
+    }, [is_account_type_modal_visible, selectAccountTypeCard]);
+
     const is_financial_available = trading_platform_available_accounts.some(
         (available_account: TTradingPlatformAvailableAccount) => available_account.market_type === 'financial'
     );
 
     const is_synthetic_available = trading_platform_available_accounts.some(
         (available_account: TTradingPlatformAvailableAccount) => available_account.market_type === 'gaming'
     );
+    const is_swapfree_available = useHasSwapFreeAccount();
 
-    const set_account_type = () =>
-        account_type_card === 'Derived'
-            ? setAccountType({ category: 'real', type: 'synthetic' })
-            : setAccountType({ category: 'real', type: 'financial' });
+    const set_account_type = () => {
+        switch (account_type_card) {
+            case localize('Derived'):
```

Comment:
```suggestion
            case localize('Derived'):
```
This might be a problem. Shouldn't use `localize` as a comparison or check in the code

Reply:
The thing is while for non-English language the juridistion modal was not giving the correct cards in the modal.
I think it can be removed as it working fine for now.

Reply:
Done